lib/UnoCore/Targets/Native/build.sh

```diff
@@ -6,10 +6,21 @@ cd "`dirname "$0"`"
 
 if ! which @(CMake) > /dev/null 2>&1; then
     echo "ERROR: Unable to find the 'cmake' command. Make sure CMake is installed and added to PATH." >&2
+#if @(MAC:Defined)
+    echo -e "\nOn macOS, you can install CMake using Homebrew:" >&2
+    echo -e "\n    brew install cmake\n" >&2
+#endif
```

Comment:
Just a thought, should we mention the same thing about chocolatey for Windows? If so, we can of course do that separately, no need to update this PR for it.

Reply:
Good though. We should, but currently `@(CMake)` is configurable and the `where` command on Windows doesn't work well with full paths. I don't think `@(CMake)` need to be configurable and if we replace it with just `cmake`, the install-hint will work well on Windows too.